CHANGES.md

```diff
@@ -133,7 +133,7 @@
 
 ## Known Issues
 
-* (Java) Current version of protobuf has [bug](https://github.com/protocolbuffers/protobuf/issues/20599) leading to incompatibilities with clients using older versions of Protobuf ([example issue](https://github.com/GoogleCloudPlatform/DataflowTemplates/issues/2191)). This issue has been seen in SpannerIO in particular.
+* (Java) Current version of protobuf has a [bug](https://github.com/protocolbuffers/protobuf/issues/20599) leading to incompatibilities with clients using older versions of Protobuf ([example issue](https://github.com/GoogleCloudPlatform/DataflowTemplates/issues/2191)). This issue has been seen in SpannerIO in particular.
```

Comment:
shall we create one Beam issue to track the fix with Beam 2.65.0 and mark this as the blocker?

Reply:
Sure, added https://github.com/apache/beam/issues/34452 - I also have the fix in progress here - https://github.com/apache/beam/pull/34451

Reply:
Let us add this beam issue here as well. Thanks.